src/labels/CustomLabels.labels

```diff
@@ -5330,6 +5402,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>CallableApi Job.Schedule Invalid Class</shortDescription>
         <value>The Apex class {0} doesn&apos;&apos;t exist or doesn&apos;&apos;t implement the Schedulable interface.</value>
     </labels>
+    <labels>
+        <fullName>CallableApiMaximumRecordsExceeded</fullName>
+        <categories>API</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>CallableApi Method Maximum Records Exceeded</shortDescription>
+        <value>{0} is the maximum number of records processed by this CallableApi method.</value>
```

Comment:
@andrewyu-salesforce - Can you give me more context about this one? What is the maximum number and what can the user do to avoid it?

Reply:
It's a limit on the number of records that can be processed in one single API request. Right now this is set to 200 so that we don't cause more than one initial Trigger run on the RD2 object. The idea is that if anyone must call this API with more than 200 records, they should be doing so from a batch job or queueable job with their RD2 records broken up into sets of 200 or less.